src/parser/parser_typedef.h

```diff
@@ -22,6 +22,9 @@ struct HSQL_CUST_LTYPE {
 
   int total_column;
 
+  // Length of the string in the SQL query string
+  int string_length;
```

Comment:
any reason this is not size_t?

Reply:
no reason in particular. i just stuck with all the other variables being `int`

Reply:
I think we can safely assume that 4 GB is sufficient for the SQL string ;)

Reply:
You didn't just make that argument :D

Reply:
:(